Assets/MRTK/Services/CameraSystem/ProjectionOverride.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using UnityEngine;
+
+#if WINDOWS_UWP
+using Windows.Perception.Spatial;
+using Windows.Graphics.Holographic;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.CameraSystem
+{
+    /// <summary>
+    /// Helper class to override the projection parameters of the HoloLens frame being presented
+    /// so that smaller details may appear more sharp.  The FOV of the HoloLens will be smaller
+    /// as a tradeoff.
+    /// </summary>
+    /// <remarks>
+    /// Instances of this class are created dynamically by <see cref="MixedRealityCameraSystem"/>.
+    /// So there is no need for an AddComponentMenu attribute.
+    /// </remarks>
+    internal class ProjectionOverride : MonoBehaviour
+    {
+        /// <summary>
+        /// When this is true, projection will be overridden on each frame
+        /// </summary>
+        public bool EnableOverride { get; set; } = false;
+
+#if WINDOWS_UWP
+        private struct InternalHoloLensFrameStructure
```

Comment:
Can you add a doccomment link to: https://docs.microsoft.com/en-us/windows/mixed-reality/develop/unity/unity-xrdevice-advanced?

I believe that link describes the data payload here (otherwise a reader could look at this and be like what is this magic?)

Reply:
Speaking of which, is the StructLayout(LayoutKind.Sequential) marker important here as well? (i.e. it's on that linked site but not here)

Reply:
Added a link to the doc, renamed the structure to be consistent with the doc, and put in the attribute.  I think Sequential is the default but it's good to be consistent with the doc.